server/src/main/java/org/apache/uniffle/server/ShuffleServerMetrics.java

```diff
@@ -69,6 +69,10 @@ public class ShuffleServerMetrics {
 
   private static final String IS_HEALTHY = "is_healthy";
   private static final String ALLOCATED_BUFFER_SIZE = "allocated_buffer_size";
+  private static final String EXPIRED_PRE_ALLOCATED_BUFFER_SIZE =
```

Comment:
these metrics should be counter rather than gauge. Right ? 

And the name should be obey the rule. xxxx_total